var/spack/repos/builtin/packages/omega-h/package.py

```diff
@@ -18,7 +18,7 @@ class OmegaH(CMakePackage):
     git      = "https://github.com/BlueBrain/omega_h.git"
 
     version('develop', branch='master')
-    version('9.31.2.dev2', sha256='08be63b68a012ed8fcf4b7a217cbaadf46faa3f3c937ec8224ff9c542eb6ae62')
+    version('9.31.3.dev1', sha256='47ccb373b4fb4cc6b56ff932badb5674b0c27ee23ad8f3d8197d49fd7678f0a6')
```

Comment:
Not a fan.

The old package is still going to hang around in the package installation database. Is there any harm in leaving it in?

Reply:
No